docs/tutorial_basics.rst

```diff
@@ -469,15 +467,8 @@ In this case, we set the :func:`set_compile_flags` hook to run before the ReFram
    The pipeline hooks, as well as the regression test pipeline itself, are covered in more detail later on in the tutorial.
 
 
-In this example, the generated executable takes a single argument which sets the number of threads that will be used.
-As seen in the previous examples, executable options are defined with the :attr:`executable_opts <reframe.core.pipeline.RegressionTest.executable_opts>` variable, and here is set to ``'16'``.
-Also, the reader may notice that this example no longer redirects the standard output of the executable into a file as the previous examples did.
-Instead, just with the purpose of keeping the :attr:`executable_opts <reframe.core.pipeline.RegressionTest.executable_opts>` simple, we use ReFrame's internal mechanism to process the standard output of the executable.
-Similarly to the parameters and the compiler settings, the output of a test is private to each of the instances of the :class:`HelloThreadedTest` class.
-So, instead of inspecting an external file to evaluate the sanity of the test, we can just set our sanity function to inspect this attribute that contains the test's standard output.
-This output is stored under :attr:`self.stdout` and is populated only after the executable has run.
-Therefore, we can set the :attr:`~reframe.core.pipeline.RegressionTest.sanity_patterns` with the :func:`set_sanity_patterns` pipeline hook that is scheduled to run before the ``sanity`` pipeline stage.
-Again, pipeline stages will be covered detail further on, so for now, just think of this ``sanity`` stage as a step that occurs after the test's executable is run.
+In this example, the generated executable takes a single argument which sets the number of threads to be used.
+The options passed to the test's executable can be set throught the :attr:`executable_opts <reframe.core.pipeline.RegressionTest.executable_opts>` variable, which in this case is set to ``'16'``.
```

Comment:
```suggestion
The options passed to the test's executable can be set through the :attr:`executable_opts <reframe.core.pipeline.RegressionTest.executable_opts>` variable, which in this case is set to ``'16'``.
```

Reply:
While `attr:executable_opts ...` link is working fine here, it does not above (line 361)

Reply:
Yeah, I'm not sure you can do that. I think the only solution would be to have a link to the config page.